native/package.json

```diff
@@ -23,7 +23,8 @@
     "react": "^16.3.1",
     "react-dom": "^16.3.1",
     "react-native": "^0.55.3",
-    "react-test-renderer": "^16.3.1"
+    "react-test-renderer": "^16.3.1",
+    "react-native-linear-gradient": "skyscanner/react-native-linear-gradient#new-gradient-implementation"
```

Comment:
Why not publish this? because it's "temporary"?

Reply:
Discussed offline